.github/workflows/ci-wordpress-rollback.yaml

```diff
@@ -75,18 +77,18 @@ jobs:
       # Adding server ip to known_hosts
       - name: Update known_hosts
         run: |
-          ssh-keyscan -H ${{ inputs.server_host }} >> ~/.ssh/known_hosts
+          ssh-keyscan -H ${{ inputs.HOST }} >> ~/.ssh/known_hosts
 
-      # Deploy the project
+      # Rollback of the release
       - name: Ansistrano rollback
         env:
           # Environment variables for ansible must be in uppercase
-          ANSIBLE_GATHERING: 'smart'
-          ANSIBLE_PIPELINING: True
-          ANSIBLE_REMOTE_USER: ${{ inputs.SSH_DEPLOY_USER }}
+          ANSIBLE_GATHERING: False # Disabling the assembly of system information to speed up the workflow
+          ANSIBLE_PIPELINING: True # We increase the performance of the pipeline (See https://docs.ansible.com/ansible/latest/reference_appendices/config.html#ansible-pipelining)
+          ANSIBLE_REMOTE_USER: ${{ inputs.USERNAME }}
```

Comment:
display comments above the lines

Reply:
for each env var include a link to official doc about such var so we can gain more info if needed.

Reply:
done